Sources/CasePaths/Operators.swift

```diff
@@ -38,6 +38,54 @@ public prefix func / <Root>(
   .case(root)
 }
 
+/// Returns a case path for the given embed function.
+///
+/// - Note: This operator is only intended to be used with enum cases that have no associated
+///   values. Its behavior is otherwise undefined.
+/// - Parameter embed: An embed function.
+/// - Returns: A case path.
+public prefix func / <Root, A, B>(
+  embed: @escaping (A, B) -> Root
```

Comment:
Because of the `CasePath<Root, Void>` overload (so that `enum Foo { case bar }` can produce case path `/Foo.bar`), these overloads make case paths with multiple associated values more specific.

Without them:

```swift
enum Foo { case bar(Int, Int) }

/Foo.bar // assumes CasePath<(Int, Int) -> Foo, Void>, not CasePath<Foo, (Int, Int)>
```